### ISSUE TITLE
Update Windows section of Install in README

### ISSUE THREAD

**Issue by gmhuestis** [GuidoBartoli/sherloq#74]
I went through first-time install in Windows environment. There are a couple of things that could be updated for installation instructions and some of these might be able to be fixed with updated requirements_win

1:
Instead of installing or using "latest Python", use Python 3.10 (version 3.10.11 was latest with Windows installer at time of writing) to be able to use TensorFlow 2.11.1. Note: I tried the current version of TensorFlow in latest Python 3.12.1 but it looked like there were going to be a lot of dependency issues to resolve so I figured using Python 3.10 was an easier and faster way to get it running since this is my first time using Sherloq.

2:
I was getting errors about libmagic and using existing fix worked for me:
```
pip uninstall python-magic python-magic-bin
pip install python-magic-bin
```

3:
Then I was getting 
ImportError: cannot import name 'convert_dict_qtables' from 'PIL.JpegImagePlugin'
as noted in Issue #71 
It appears that convert_dict_qtables was removed from Pillow v10 so I used the following to fix
```
pip uninstall pillow
pip install pillow==9.5
```

Those changes got me up and running quickly under Windows 10 and also Windows 11.
Thanks for great tool!

**Comment by GuidoBartoli** [comment]
Fixed in 2c7a07c, thanks!

**Comment by TheMrRight** [comment]
Had this Exact Problem, THANKS A LOT ❤️
